### ISSUE TITLE
checkpoint release

### ISSUE THREAD

**Issue by yk-pku** [LingyiHongfd/LVOS-api#2]
Hi, could you release your checkpoints for the finetuned STCN, RDE, and XMem, as I got worse results after fine-tuning. I followed the fine-tuning instructions in paper (2 epochs and 5e-4 learning rate).

**Comment by yk-pku** [comment]
I would greatly appreciate it if you could disclose more fine-tuning details~

**Comment by yk-pku** [comment]
Hi, I found that changing the size of training data to 480P, which is the same as YouTube-VOS and DAVIS, can lead to much better results. Now, I can get almost the same results reported by paper.

**Comment by LingyiHongfd** [comment]
Cool! Sry for late response. I'm occupied with another project in these days. 
I can't find these checkpoints because it's been so long. I think that you don't have to limit yourself to the fine-tuning only on LVOS. Many recent works such as Cuite, OneVOS, and SAM2, have been evaluated on LVOS, and these methods are trained on the combination of many VOS datasets including LVOS.
If you have any question, feel free to let me know.

**Comment by yk-pku** [comment]
Thanks for your suggestion~
